### ISSUE TITLE
`asana` run logs don't print error message

### ISSUE THREAD

**Issue by aleksa-krolls** [OpenFn/adaptors#530]
## Request
Update the Asana adaptor to print the error message returned by the Asana API. Right now it shows up as `data: { errors: [Array] }`... but I want to see the message that's in that array! 

This is not being done across functions. For example, [see L342 in this run failure](https://v1.openfn.org/projects/consosci/runs/0662743b-6133-7bb6-ac2d-d0db76125123). 
```
AxiosError: Request failed with status code 401
data: { errors: [Array] }
```

## To test
Run [this Asana job](https://github.com/OpenFn/ConSoSci/blob/master/form-sharing/4-upsertAsanaTask.js) with a fake personal access token as the credential to reproduce the `401` error example above.

**Comment by josephjclark** [comment]
Hey @aleksa-krolls is this blocking anything? It's fine on V2 with the latest adaptor. Fixing an old adaptor on v1 is non trivial.

We can do it, but it's gotta be worth the effort

**Comment by aleksa-krolls** [comment]
If the recommendation is just to update the adaptor version, then that's fine. (That wasn't clear before.)

**Comment by josephjclark** [comment]
Platform needs updating to support the latest adaptor version. There are breaking changes so job might need migrating. And I'm not 100% it'll fix the issue on v1 anyway - can't test to see!

**Comment by aleksa-krolls** [comment]
I've suggested a very quick v1 test [on this slack thread](https://openfn.slack.com/archives/C017ELVRSM8/p1722243955790539) to see. But if it's going to be a lot of work, then forget it and we can look to migrate the client's jobs later

**Comment by josephjclark** [comment]
@aleksa-krolls I'm afk right now but I'll check the slack thread.

Thinking about it, I _think_ we could put out a 2.x patch with a quick fix and load that onto platform 

Back at my desk an in hour and I'll look closely at the options

**Comment by josephjclark** [comment]
I've released a patch build 2.1.7. Once that's been released on platform I hope it'll close the issue
